From c6142e920fc53de3a5d97e73f6cfe9a62c738964 Mon Sep 17 00:00:00 2001
From: Rex Fu <Rex.Fu@amd.com>
Date: Fri, 26 Jun 2026 17:07:14 +0800
Subject: [PATCH] arm: dts: anacapa: Align leakage SGPIO pin names

Summary:

* Align leakage SGPIO pin names with upstream Anacapa DTS.

LORE: https://lore.kernel.org/all/20260602-anacapa-devlop-phase-devicetree-v3-2-7c93c5df8d9b@gmail.com/

Motivation:

* To keep local Anacapa DTS naming consistent with upstream DTS.

Test Plan:

* Build pass

Signed-off-by: Rex Fu <Rex.Fu@amd.com>
---
 .../aspeed/aspeed-bmc-facebook-anacapa-evt2.dts    | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-anacapa-evt2.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-anacapa-evt2.dts
index e5d8b5e71859..92aa58735868 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-anacapa-evt2.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-anacapa-evt2.dts
@@ -1020,7 +1020,7 @@ &sgpiom0 {
 	"Channel0_leakage_EAM3", "FM_CPU_FPGA_JTAG_MUX_SEL",
 	"Channel1_leakage_EAM0", "FM_SCM_JTAG_MUX_SEL",
 	"Channel2_leakage_Manifold1", "FM_BRIDGE_JTAG_MUX_SEL",
-	"Channel3_leakage", "FM_CPU0_NMI_SYNC_FLOOD_N",
+	"Channel5_leakage_present", "FM_CPU0_NMI_SYNC_FLOOD_N",
 	"Channel4_leakage_Manifold2", "BMC_AINIC0_WP_R2_L",
 	"Channel5_leakage_EAM1", "BMC_AINIC1_WP_R2_L",
 	"Channel6_leakage_CPU_DIMM", "CPLD_BUF_R_AGPIO330",
@@ -1030,19 +1030,19 @@ &sgpiom0 {
 	"RSVD_RMC_GPIO3", "RTM_MUX_L",
 	"LEAK_DETECT_RMC_N", "RTM_MUX_R",
 	"HDR_P0_NMI_BTN_BUF_R_N", "FPGA_JTAG_SCM_DBREQ_N",
-	"No_Leak_Sensor_flag", "whdt_sel",
+	"Channel6_leakage_present", "whdt_sel",
 	"", "",
 	"", "",
 	"", "",
 	"", "",
 
 	/* D0-D7 line 48-63 */
-	"PWRGD_CHAD_CPU0_FPGA", "",
-	"PWRGD_CHEH_CPU0_FPGA", "",
-	"PWRGD_CHIL_CPU0_FPGA", "",
-	"PWRGD_CHMP_CPU0_FPGA", "",
+	"Channel0_leakage_present", "",
+	"Channel1_leakage_present", "",
+	"Channel2_leakage_present", "",
+	"Channel4_leakage_present", "",
 	"AMC_BRD_PRSNT_CPLD_L", "",
-	"", "",
+	"Channel7_leakage_present", "",
 	"", "",
 	"", "",
 
-- 
2.53.0-Meta

